src/nrnpython/nrnpython.cpp

```diff
@@ -45,16 +50,26 @@ int nrnpy_site_problem;
 
 void nrnpy_augment_path() {
   static int augmented = 0;
-  char buf[1024];
   if (!augmented && strlen(neuronhome_forward()) > 0) {
-    // printf("augment_path\n");
     augmented = 1;
-    sprintf(buf, "sys.path.append('%s/lib/python')", neuronhome_forward());
     int err = PyRun_SimpleString("import sys");
     assert(err == 0);
-    err = PyRun_SimpleString(buf);
-    assert(err == 0);
-    sprintf(buf, "sys.path.prepend('')");
+#if defined(__linux__) || defined(DARWIN)
+    // If /where/installed/lib/python/neuron exists, then append to sys.path
+    std::string lib = std::string(path_prefix_to_libnrniv());
+    // For an autotools build, it ends with x86_64/lib/
+    size_t pos = lib.find("x86_64/lib/", lib.length() - 11);
```

Comment:
`x86_64 ` is hardcoded here? what about other architectures? e.g. on Marconi100, we get powerpc64le etc

Reply:
Ugh. This is why reviews are so useful. Thanks!